### PR TITLE
feat(map): add `create_init_cert_subcmd` var

### PR DESCRIPTION
**Pull request by sticky-note**
### PR progress checklist (to be filled in by reviewers)

- [ ] Changes to documentation are appropriate (or tick if not required)
- [ ] Changes to tests are appropriate (or tick if not required)
- [ ] Reviews completed

---

#### Primary type

- [ ] `[build]`    Changes related to the build system
- [ ] `[chore]`    Changes to the build process or auxiliary tools and libraries such as documentation generation
- [ ] `[ci]`       Changes to the continuous integration configuration
- [x] `[feat]`     A new feature
- [ ] `[fix]`      A bug fix
- [ ] `[perf]`     A code change that improves performance
- [ ] `[refactor]` A code change that neither fixes a bug nor adds a feature
- [ ] `[revert]`   A change used to revert a previous commit
- [ ] `[style]`    Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc.)

#### Secondary type

- [ ] `[docs]`     Documentation changes
- [ ] `[test]`     Adding missing or correcting existing tests

### Does this PR introduce a `BREAKING CHANGE`?

No.

### Describe the changes you're proposing

Add a `create_init_cert_subcmd` var that can be filled by config/pillar for specific use case

### Documentation checklist

- [ ] Updated the `README` (e.g. `Available states`).
- [x] Updated `pillar.example`.

### Testing checklist

- [ ] Included in Kitchen (i.e. under `state_top`).
- [ ] Covered by new/existing tests (e.g. InSpec, Serverspec, etc.).
- [ ] Updated the relevant test pillar.

### CODE REVIEW

**Comment by myii** [comment]
@sticky-note Please don't request a review from `@saltstack-formulas/contributors` because there's been a valid complaint about this:

* https://github.com/orgs/saltstack-formulas/teams/contributors/discussions/3

This formula has a dedicated codeowner in @javierbertoli so best to wait for his feedback (or ping him specifically if there's a substantial delay).

Thanks.

File: letsencrypt/defaults.yaml
Comment (javierbertoli) [inline letsencrypt/defaults.yaml]: Typo, extra `o` in certonly. Can you fix it, please?

File: letsencrypt/domains.sls
Comment (javierbertoli) [inline letsencrypt/domains.sls]: Typo, extra `x` after the `else` clause, please remove it.

**Review by javierbertoli — CHANGES_REQUESTED** [review]
Hi @sticky-note. There are a couple of typos, one of which will make the formula fail. If you fix them, I'll be glad to merge the PR.

File: letsencrypt/domains.sls
Comment (daks) [inline letsencrypt/domains.sls:45]: It seems there is no default value for `create_init_cert_subcmd`, what does happen here if it's not set in pillar?

Comment (myii) [reply letsencrypt/domains.sls:45]: @daks The default is supplied but needs to be fixed, as mentioned above:

* https://github.com/saltstack-formulas/letsencrypt-formula/pull/77/files#diff-85fe05632fbab67e7af1ce552c2237b89d2fe6515d769a76639f85c62c64a745R34

I.e. `certoonly` => `certonly` -- https://github.com/saltstack-formulas/letsencrypt-formula/pull/77#discussion_r511824324.

Comment (sticky-note) [reply letsencrypt/domains.sls:45]: fixed

**Comment by javierbertoli** [comment]
Thanks, @sticky-note !

**Pull request merged**
